frontend/web/components/modals/CreateRole.tsx

```diff
@@ -373,6 +371,54 @@ const CreateRole: FC<CreateRoleType> = ({
           id='description'
           placeholder='E.g. Some role description'
         />
+        <FormGroup className='mb-5 setting'>
+          <InputGroup
+            title={'Tags'}
+            unsaved={isEdit && roleTagsChanged}
+            component={
+              <>
+                <p className='fs-small lh-sm'>
+                  Select the project where you want to manage your tags.{' '}
+                  <Button
+                    theme='text'
+                    target='_blank'
+                    href='https://docs.flagsmith.com/system-administration/rbac#add-tags-to-a-role'
+                    className='fw-normal'
+                  >
+                    Learn more.
+                  </Button>
+                </p>
+                <div className='mb-2' style={{ width: 250 }}>
+                  <ProjectFilter
+                    organisationId={organisationId!}
+                    onChange={(p) => {
+                      setProject(p)
+                      getTags(
```

Comment:
Is there a reason why this is called? seems wrong.

Reply:
Looking at this more closely, it does look like this is unnecessary. AddEditTags uses useGetTagsQuery.

Reply:
You're right, removed.